gtk4/src/print_job.rs

```diff
@@ -6,20 +6,23 @@ use std::boxed::Box as Box_;
 
 impl PrintJob {
     #[doc(alias = "gtk_print_job_send")]
-    pub fn send<P: Fn(&PrintJob, &glib::Error) + 'static>(&self, callback: P) {
+    pub fn send<P: Fn(Result<(), glib::Error>) + 'static>(&self, callback: P) {
         let callback_data: Box_<P> = Box_::new(callback);
-        unsafe extern "C" fn callback_func<P: Fn(&PrintJob, &glib::Error) + 'static>(
-            print_job: *mut ffi::GtkPrintJob,
+        unsafe extern "C" fn callback_func<P: Fn(Result<(), glib::Error>) + 'static>(
+            _print_job: *mut ffi::GtkPrintJob,
             user_data: glib::ffi::gpointer,
             error: *const glib::ffi::GError,
         ) {
-            let print_job = from_glib_borrow(print_job);
-            let error = from_glib_borrow(error);
+            let result = if error.is_null() {
+                Ok(())
+            } else {
+                Err(from_glib_none(error))
+            };
             let callback: &P = &*(user_data as *mut _);
-            (*callback)(&print_job, &error);
+            (*callback)(result);
         }
         let callback = Some(callback_func::<P> as _);
-        unsafe extern "C" fn dnotify_func<P: Fn(&PrintJob, &glib::Error) + 'static>(
+        unsafe extern "C" fn dnotify_func<P: Fn(Result<(), glib::Error>) + 'static>(
```

Comment:
```suggestion
        unsafe extern "C" fn dnotify_func<P>(
```

should be enough

Reply:
I kept it as it was before and mechanically replaced... I will wait for CI to pass before applying this since I cannot test locally

Reply:
The CI now should be green, so feel free to rebase